doc/ikfast_tutorial.rst

```diff
@@ -38,26 +38,12 @@ OpenRAVE Installation
 ----------------------
 **Binary Install**
 
-OpenRave is available for Ubuntu up to version 12.04 using these commands::
+ sudo add-apt install ros-indigo-openrave
```

Comment:
- apt-get works only for indigo, meaning 14.04 http://rosindex.github.io/p/openrave/#indigo
- at this moment, we release only indigo, so `indigo` is ok, not so difficult to release on jade, kinetic
- how about `Binary package of OpenRave from ROS repository is available for Ubuntu  version 14.04 using these commands`